recipe/cap_mpjpeg_decoder.patch

```diff
@@ -0,0 +1,11 @@
+--- modules/videoio/src/cap_mjpeg_decoder.cpp
++++ modules/videoio/src/cap_mjpeg_decoder.cpp
+@@ -790,7 +790,7 @@
+     result.reserve(chunk.m_size);
+     result.resize(chunk.m_size);
+ 
+-    m_file_stream.read(result.data(), chunk.m_size);
++    m_file_stream.read(&result.front(), chunk.m_size);
```

Comment:
[`mpjpeg_decoder` removed in latest version](https://github.com/Itseez/opencv/tree/master/modules/videoio/src)